src/stylesheets/_typography.scss

```
@@ -1,14 +1,69 @@
+@use "src/stylesheets/responsiveness";
```

Remember that the root is now the `/src` folder so all the imports will happen from that folder. 

```suggestion
@use "~stylesheets/responsiveness";
```